backend/open_webui/routers/evaluations.py

```
@@ -94,12 +96,7 @@ async def delete_all_feedbacks(user=Depends(get_admin_user)):
 @router.get("/feedbacks/all/export", response_model=list[FeedbackModel])
 async def get_all_feedbacks(user=Depends(get_admin_user)):
```

Ruff (0.12.2)

Redefinition of unused `get_all_feedbacks` from line 75

(F811)